sql/moz-fx-data-shared-prod/contextual_services_derived/event_aggregates_spons_tiles_v1/query.sql

```diff
@@ -4,7 +4,7 @@ WITH combined AS (
     DATE(submission_timestamp) AS submission_date,
     'desktop' AS form_factor,
     normalized_country_code AS country,
-    LOWER(advertiser) AS advertiser,
+    lower(REPLACE(lower(advertiser), "o=45:a", "yandex")) AS advertiser,
```

Comment:
You don't need to lowercase twice, this should work

```suggestion
    REPLACE(LOWER(advertiser), "o=45:a", "yandex") AS advertiser,
```